app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss

```diff
@@ -136,6 +136,32 @@
   &__divider{
     margin-bottom: 0;
   }
+
+  &--disabled{
+    input[type="radio"]:checked + label{
+      background-color: lighten($sky, 4%);
+      border: 2px solid lighten(#b4e2f7, 12%);
```

Comment:
can we use a variable for this hex code? can you lighten $sky a bit to achieve the same thing?